### ISSUE TITLE
Auto-adjust actuation voltage based on feedback (Trac #50)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#54]
The current implementation assumes a constant, hard-coded amplifier gain; however, in reality, gain varies with frequency and in some cases, load. The software should be able to measure the output voltage from the amplifier and dynamically adjust to achieve the target voltage.

Note that this could also include correcting for the feedback voltage, which slightly reduces the voltage drop across the device.

Migrated from http://microfluidics.utoronto.ca/ticket/50
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:43", 
    "description": "The current implementation assumes a constant, hard-coded amplifier gain; however, in reality, gain varies with frequency and in some cases, load. The software should be able to measure the output voltage from the amplifier and dynamically adjust to achieve the target voltage.\n\nNote that this could also include correcting for the feedback voltage, which slightly reduces the voltage drop across the device.", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763583719301", 
    "component": "microdrop", 
    "summary": "Auto-adjust actuation voltage based on feedback", 
    "priority": "minor", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-02-20T03:53:40", 
    "milestone": "Microdrop 1.0", 
    "owner": "ryan", 
    "type": "enhancement"
}
```

**Comment by cfobel** [comment]
Trac update at 20120624T05:25:43: 
* ryan commented:

> This is fixed in dmf_control_board commit 7dbbd92499dfa945861a2af3970eb828421ea416.


* ryan changed resolution from "" to "fixed"
* ryan changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:43: ryan changed component from "hardware/communication" to "microdrop"
